assets/findify-filter.css

```diff
@@ -339,12 +339,6 @@ button {
   .findify-filters--filter-header {
     justify-content: unset;
   }
-
-  /* Ellipsis for mobile, increase with maximum */
-  .findify-filters-checkbox-item-value > span,
-  .findify-filters--checkbox-nested-value > span {
-    width: 250px;
-  }
 }
```

Comment:
@rozhnovaYana Here you have removed ellipsis, why?
This will make sure that filters name if too long will be cut and display ...